### PR TITLE
Fixes #22 #13 - footer module & documentation

### LINKED ISSUE #22: add footer documentation

### PR DESCRIPTION
**Pull request by zoepage**
- [x] module
- [x] documentation

### CODE REVIEW

**Comment by zoepage** [comment]
@magsout Would you like to review this PR? ;)

**Comment by zoepage** [comment]
Also @kotzendekrabbe <3

File: docs/css/footer.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#footer {
```
Comment (kotzendekrabbe) [inline docs/css/footer.css]: I would say, remove the id footer here, because it's empty you don't need it, right?

File: docs/css/footer.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+.footer-item-link {
+	text-decoration: none;
+}
+
+.active .footer-item-link,
+.footer-item-link:hover {
+	text-decoration: underline;
+}
+
+#footer .active .footer-item-link {
```
Comment (kotzendekrabbe) [inline docs/css/footer.css]: do you need the ID at this point? If not, maybe delete it - it's a high css specificity

File: docs/css/footer.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	text-align: right;
+	margin-top: auto;
+}
+
+.footer-icon  {
+	display: inline-block;
+	padding: 5px 20px;
+}
+
+.icon-twitter .icon {
```
Comment (kotzendekrabbe) [inline docs/css/footer.css:46]: could also be only icon-twitter and move the class at the html to the svg class="icon icon-twitter".

**Review by kotzendekrabbe — COMMENTED** [review]
a few little things :)
rest looks fine to me.

File: docs/css/footer.css (hunk 3 above)
Comment (zoepage) [reply docs/css/footer.css:46]: The `.icon` holds definitions that `.icon-twitter` is inheriting, which is the default definition for the icons.

File: docs/css/footer.css (hunk 1 above)
Comment (zoepage) [reply docs/css/footer.css]: Yeah, you are right.

**Review by kotzendekrabbe — APPROVED** [review]

**Pull request merged**